packages/yourloops/lib/http/axios.service.ts

```diff
@@ -45,15 +59,9 @@ export const onFulfilled = async (config: AxiosRequestConfig): Promise<AxiosRequ
       }
     }
   }
-  return config
 }
 
-function initAxios(): void {
-  axios.defaults.baseURL = appConfig.API_HOST
-  /**
-   * We use axios request interceptor to set the access token into headers each request the app send
-   */
-  axios.interceptors.request.use(onFulfilled)
-}
+const internalAxios = new AxiosService().init(appConfig.API_HOST, true)
+const auth0Axios = new AxiosService().init(`https://${appConfig.AUTH0_DOMAIN}`)
```

Comment:
I don't really see the necessity to have two different instances of Axios. I think it could be confusing, knowing that we call only one auth0 api. The baseUrl can be overrided with the API call in axios params and concerning the headers to be removed, there is already a trick with `parmas.noHeader` that works ^^
What do you think, what was your motivation ?

Reply:
Changes reverted